src/services/public_http_server/handlers/relay_webhook/error.rs

```diff
@@ -13,9 +13,18 @@ use {
 
 #[derive(Debug, thiserror::Error)]
 pub enum RelayMessageClientError {
-    #[error("Received wc_notifyWatchSubscriptions (4010) on wrong topic: {0}")]
+    #[error("Received 4010 on wrong topic: {0}")]
     WrongNotifyWatchSubscriptionsTopic(Topic),
 
+    #[error("Received 4008 on unrecognied topic: {0}")]
+    WrongNotifyUpdateTopic(Topic),
+
+    #[error("Received 4004 on unrecognied topic: {0}")]
```

Comment:
ditto

Reply:
Fixed in ec80ea33f7970437604885f9ab182d9116c02463